MySQL CRUD/conn.js

```diff
@@ -0,0 +1,13 @@
+import mysql from 'mysql';
+
+export var conn = mysql.createConnection({
+    host: "localhost",
+    user:"root",
+    password: "Admin@123",
+    database: "userdb",
+    multipleStatements: true
+})
+conn.connect((err) => {
+    if (!err) console.log("Connected!"); 
+    else console.log('Db connection failed \n Error: '+ JSON.stringify(err, undefined, 2));

```

Comment:
Use `if` with `return` statement

```suggestion 
    // Handle error first
    if (err) {
     console.error('Db connection failed \n Error: '+ JSON.stringify(err, undefined, 2));
     return;
     } 
     
     // Your logic to handle success case
     console.log("Connected!");
```